akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -559,7 +559,7 @@ object Link extends ModeledCompanion[Link] {
   def apply(values: LinkValue*): Link = apply(immutable.Seq(values: _*))
   implicit val valuesRenderer = Renderer.defaultSeqRenderer[LinkValue] // cache
 }
-final case class Link(values: immutable.Seq[LinkValue]) extends jm.headers.Link with RequestResponseHeader {
+final case class Link(values: immutable.Seq[LinkValue]) extends jm.headers.Link with ResponseHeader {
```

Comment:
Especially this one I found hard to see from the RFC if it actually is ok as request header, but couldn't find any references online to anyone using it in requests so I changed it.